pkg/model/image_manager.go

```diff
@@ -22,49 +24,72 @@ const (
 	DefaultRHSSOInitContainer    = "registry.redhat.io/rh-sso-7-tech-preview/sso74-init-container-rhel8:7.4"
 	DefaultRHMIBackupContainer   = "quay.io/integreatly/backup-container:1.0.16"
 	DefaultPostgresqlImage       = "registry.access.redhat.com/rhscl/postgresql-10-rhel7:1"
+
+	keycloakImagePullSecret              = "KEYCLOAK_IMAGE_PULL_SECRET"
```

Comment:
These aren't exported from package to avoid confusion and are not used as keys in `Images` map. Users would get secret for Keycloak by doing `Images[KeycloakImage].ImagePullSecret`